server/helpers/verifyEmailMessage.js

```diff
@@ -0,0 +1,13 @@
+const verifyEmailMessage = {
+  subject: 'Welcome to Author\'s Haven! Please Confirm your email',
+  body:
```

Comment:
Nice work. Can you please try to add styles to the email so that it is not just plain. Use inline styles e.g. 

```
<div style="height: 20em, background-color: #E6FFED; border: 1px solid black; padding: 0.5em;">
<p><a href="url" style="color: #E6FFED; background-color: white; padding: 1em 2em ">CLICK HERE!</a> to verify your email</p>
</div>
```

This is just an example of what I mean. You can design it on your browser and even add an image

Reply:
@Darthrighteous and @KvNGCzA feedback received. Your suggestions are necessary. I am working on them. Get back to you soon